### ISSUE TITLE
merge PR#81 broke MSVC compatibility

### ISSUE THREAD

**Issue by ARF1** [Blosc/c-blosc#95]
The merge of PR #81 broke MSVC compatibility (at least when used with bcolz). MSVC complains of:

```
blosclz.obj : error LNK2019: unresolved external symbol _llabs referenced in fun
ction _blosclz_decompress
```

The culprit seems to be that the PR replaced `abs(op-ref)` with `llabs(op-ref)` in `blosc\blosclz.c` which seems to be undefined for MSVC. The problem persists for the current tip.

**Comment by FrancescAlted** [comment]
Ok, so apparently Microsoft introduced `llabs` recently (MSVC 18.0 ?).  Which MSVC are you using?  I suppose the correct fix is to add a check in MSVC version for using `abs` or `llabs`.

**Comment by ARF1** [comment]
I am using MSVC 2008 since apparently that is what Pyhon 2.7 is compiled 
with and thus avoids runtime issues.

```
C:\>"C:\Program Files\Common Files\Microsoft\Visual C++ for 
Python\9.0\VC\Bin\cl.exe"
Microsoft (R) 32-bit C/C++ Optimizing Compiler Version 15.00.30729.01 
for 80x86
Copyright (C) Microsoft Corporation.  All rights reserved.
```

**Comment by FrancescAlted** [comment]
Ok, so the fix in rev 8250362 works for you?

**Comment by ARF1** [comment]
Works for me. Thank you.
